tasks/pull.yaml

```diff
@@ -0,0 +1,23 @@
+variables:
+  - name: FLAVOR
+    default: upstream
+  - name: TARGET_REPO
+    default: ghcr.io/defenseunicorns/packages/uds
+
+tasks:
+  - name: latest-package-release
+    inputs:
+      path:
+        description: Path relative to the repositories root where the package needs to go
+        default: .
+    actions:
+      - description: Get the current Zarf package name
+        cmd: cat zarf.yaml | yq .metadata.name
```

Comment:
wondering if there is a different way to do this that doesn't require `yq` perhaps?